### DISCUSSION TITLE
Missing clear roadmap for each repos and mailing list

### DISCUSSION THREAD

**Discussion by johnnykeats** [bluebanquise/bluebanquise#643] [Ideas]
Hello,

BlueBanquise project is growing, and is now split between multiple repos.
A coherent roadmap should be provided, including in one file targets for each repository.

A mailing list should also be created, to inform about releases, events, security issues, ...

Any opinion on this?

**Comment by oxedions** [comment]
Hello JK,

Yes, at least a mailing list could be interesting I agree.
I need to check with OVH if the bluebanquise domain can also host a mail (currently it is only redirection).

Regarding roadmap, well difficult as things pop up on the flow, but this is something I agree we should stabilize.

**Comment by btravouillon** [comment]
Is the Slack workplace deprecated?

**Reply by oxedions** [reply]
Kind of yes 😐 
Most new devs requested to use Discord, so we switched on a Discord server, but it was not official, just a quick communication channel for urgent fix during 1.5 and 1.6 devs. Most of the time, we have direct P2P exchanges on Discord. Let me send you an invitation link.
